root/static/scripts/edit/URLCleanup.js

```diff
@@ -1831,8 +1852,20 @@ const CLEANUPS = {
         switch (id) {
           case LINK_TYPES.streamingfree.recording:
           case LINK_TYPES.streamingfree.release:
+            if (prefix === 'user') {
+              return {
+                error: linkToVideoMsg,
```

Comment:
linkToVideoMsg says "artist, label, etc.", but for some reason, niconico is restricted to `LINK_TYPES.videochannel.artist`. Do we know if this is intentional? Are labels, places, etc. not allowed to have a channel there?

Reply:
I did not find anything about it in MBS-9944. Possibly niconico is just for artists, not sure.

By the way, you might want to check MBS-10412 again.

Reply:
Made some changes to niconico (MBS-10831 / MBS-10412) and put them at the end of the PR, do check :)

Reply:
Nitpicking the message of commit https://github.com/metabrainz/musicbrainz-server/pull/1515/commits/75ccb1670a00890441fd5c6dd2272826960d9d7a which seems erroneous:
* It should probably be about user links, not “channel” links.
* “videochannel entities” should probably be replaced with “videochannel rels”.